.github/workflows/deploy.yml

```diff
@@ -21,7 +21,7 @@ jobs:
             | sed s/{CLOUDFLARE_ACCT_ID}/${{ secrets.CLOUDFLARE_ACCT_ID }}/ \
             | sed s/{STAMPY_KV_ID}/${{ secrets.STAMPY_KV_ID }}/ \
             | sed s/{CODA_TOKEN}/${{ secrets.CODA_TOKEN }}/ \
-            | sed s/{CODA_INCOMING_TOKEN}/${{ secrets.CODA_INCOMING_TOKEN }}/ \
+            | sed s/{CODA_WRITE_TOKEN}/${{ secrets.CODA_WRITE_TOKEN }}/ \
```

Comment:
these will have to be changed :/ The previous tokens only allowed writing to the incoming table. An alternative would be to add a new view with just the metrics (for lack of a better word), which should be a lot safer at the cost of extra tokens

Reply:
ok, you now have Admin access to be able to add more secrets to https://github.com/StampyAI/stampy-ui/settings/secrets/actions